src/test/cli-integration/rbd/formatted-output.t

```diff
@@ -550,15 +550,15 @@ whenever it is run. grep -v to ignore it, but still work on other distros.
     <name>quuy</name>
   </images>
   $ rbd list -l
-  NAME       SIZE PARENT FMT PROT LOCK 
-  foo       1024M          1           
-  foo@snap  1024M          1           
-  quux      1024k          1      excl 
-  bar       1024M          2           
-  bar@snap   512M          2 yes       
-  bar@snap2 1024M          2           
-  baz       2048M          2      shr  
-  quuy      2048M          2           
+  NAME         SIZE PARENT FMT PROT LOCK
```

Comment:
No need to cherry-pick this file since it isn't used for client-upgrade tests (and it will break Luminous runs).

Reply:
I believe then the changeset in this file and CloneRequest can be entirely dropped, which leaves only with whatever we need to do in import/exporT?